l10n_it_fringe_benefit/models/fleet.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+#
+##############################################################################
+#
+#    Author(s): Alessandro Camilli (alessandrocamilli@openforce.it)
+#               Andrea Colangelo (andreacolangelo@openforce.it)
+#
+#    Copyright © 2016 Openforce di Camilli Alessandro (www.openforce.it)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see:
+#    http://www.gnu.org/licenses/agpl-3.0.txt.
+#
+##############################################################################
+
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+import base64
+import csv
+
+
+class fleet_vehicle_model(models.Model):
+    _inherit = 'fleet.vehicle.model'
+
+    @api.multi
+    @api.depends('fringe_benefit_ids')
+    def _compute_fb_cost_km(self):
+        for vm in self:
+            domain = [('model_id', '=', vm.id)]
+            fb = self.env['fleet.fringe.benefit'].search(
+                domain, order='date desc', limit=1)
+            vm.fringe_benefit_cost_km = fb.cost_km or False
+
+    fringe_benefit_ids = fields.One2many('fleet.fringe.benefit',
+                                         'model_id',
+                                         string='Fringe Benefit')
+    fringe_benefit_cost_km = fields.Float(string='Fringe Benefit Cost Km',
+                                          compute="_compute_fb_cost_km",
+                                          store=True)
+
+
+class fleet_fringe_benefit_version(models.Model):
+    _name = 'fleet.fringe.benefit.version'
+
+    date = fields.Date(string='Date', required=True)
+    name = fields.Char(string='Name', required=True)
```

Comment:
string parameter is useless for date and name fields. It will be created by Odoo